ConfigService/Configurations/EDDIConfiguration.cs

```diff
@@ -43,6 +43,11 @@ public class EDDIConfiguration : Config
         [JsonProperty("MainWindowPosition")]
         public Rect MainWindowPosition { get; set; }
 
+        // Hotkeys
+
+        [ JsonProperty( "Hotkeys" ) ]
+        public Dictionary<string, string> Hotkeys { get; set; } = new Dictionary<string, string>();
```

Comment:
Making a mutable collection member public never ends well, because client code will stick its fingers in and find ways to mutate it without this class knowing anything about it. This then defeats PropertyChanged mechanisms etc, which turns out to be why the config wasn't being written to disk when expected.

Better to keep the collection private and write public methods for adding and removing items. This has the additional benefit that you can change the implementation without affecting client code.